src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -317,7 +317,7 @@ public async Task Terminate(string connectionStringKey, string taskHubName, stri
             }
         }
 
-        public static dynamic DeserializeInstanceInput(string input)
+        public static string RetrieveCommandInputData(string input)
```

Comment:
Why was this change needed? I think the previous version was actually correct (parsing the contents of the file and returning it as either a `dynamic` or an `object`). Otherwise this CLI command will only support string inputs but not numbers, arrays, or complex types.